src/parser/util.js

```diff
@@ -86,3 +87,25 @@ pp.unexpected = function (pos, messageOrType = "Unexpected token") {
   }
   this.raise(pos != null ? pos : this.state.start, messageOrType);
 };
+
+pp.expectPlugin = function (name) {
+  if (!this.hasPlugin(name)) {
+    const pluginInfo = plugins[name];
+    const pointers = [];
+    if (pluginInfo) {
+      if (pluginInfo.babelSyntaxPlugin) {
+        pointers.push(`babel-plugin-${pluginInfo.babelSyntaxPlugin}`);
+      }
+      if (pluginInfo.babelTransformPlugins) {
```

Comment:
Yeah we do have syntax plugins for every transform (otherwise we would just remove the syntax plugins entirely)